src/menu/menu.c

```diff
@@ -115,6 +115,10 @@ static void menu_init (boot_params_t *boot_params) {
         __boot_tvtype = TV_NTSC;
     }
 
+    if (menu->settings.sound_enabled) {
+        sound_init_sfx();
+    }
```

Comment:
Minor nitpick because changing this setting live will not enable sounds and will require complete reboot of the menu. It's okay for now because we don't have a settings editor yet.